lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -492,6 +502,14 @@ deriveRewardAccount pwd rootPrv =
     let accPrv = deriveAccountPrivateKey pwd rootPrv minBound
     in deriveAddressPrivateKey pwd accPrv MutableAccount minBound
 
+hashVerificationKey
+    :: WalletKey k
+    => KeyRole
+    -> k d XPub
```

Comment:
```suggestion
    -> key depth XPub
```

Reply:
Funnily enough I wanted to do this originally but LSP was giving me errors. Seems to be fine now :+1:

Reply:
Fixed